Formula/hledger.rb

```diff
@@ -45,18 +45,30 @@ def install
     (buildpath/"../hledger-web").install resource("hledger-web")
     cd ".." do
       system "stack", "update"
-      system "stack", "init", "--resolver=lts-16.12"
+      File.open("stack.yaml", "w") do |f|
+        f.write <<~EOS
```

Comment:
```suggestion
      (buildpath/"stack.yaml").write <<~EOS
```
This should also work, no?

Reply:
Thanks. How would I tell it ./stack.yaml ?

Reply:
Edited my suggestion. Try that. But no longer sure; not sure why these things are happening in the parent of `buildpath`.

Reply:
stack expects to see all four haskell packages - hledger-1.20 (buildpath) and hledger-lib, hledger-ui, hledger-web - as subdirectories. For now at least.

I have a mystery problem preventing me from retesting; I'll come back to this.